bundles/org.openhab.binding.miio/README.md

```diff
@@ -2663,7 +2663,7 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 |----------------------|----------------------|------------------------------------------|------------|
 | power                | Switch               | Power                                    |            |
 | brightness           | Dimmer               | Brightness                               |            |
-| ambientBrightness    | Number               | Ambient Brightness                       |            |
+| ambientBrightness    | Dimmer               | Ambient Brightness                       |            |
```

Comment:
What does this change have to do with this PR?

Reply:
Nothing, 
It seems this was missed in the previous run, so it was picked up in the new run